src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -420,3 +420,64 @@ def load_arguments(self, _):
         c.argument(
             'service_name', options_list=['--service-name', '-n'],
             help="The name of the soft deleted API Management service instance.")
+
+    with self.argument_context('apim graphql api resolver create') as c:
```

Comment:
The command signature here does not match the one in the help message

> helps['apim graphqlapi resolver create'] = """

Reply:
In addition, can we remove the `api` section to simplify this command signature?

Reply:
did